libraries/AP_InertialSensor/AP_InertialSensor_Logging.cpp

```diff
@@ -151,21 +151,38 @@ bool AP_InertialSensor::BatchSampler::Write_ISBD() const
 // @Field: NF10: dynamic harmonic notch centre frequency for motor 10
 // @Field: NF11: dynamic harmonic notch centre frequency for motor 11
 // @Field: NF12: dynamic harmonic notch centre frequency for motor 12
+
+// @LoggerMessage: FTNS
+// @Description: Filter Tuning Message
+// @Field: TimeUS: microseconds since system startup
+// @Field: I: instance
+// @Field: NF: dynamic harmonic notch centre frequency
+
 void AP_InertialSensor::write_notch_log_messages() const
 {
     for (auto &notch : harmonic_notches) {
         const uint8_t i = &notch - &harmonic_notches[0];
-        if (!notch.params.enabled()) {
+        if (!notch.params.enabled() || (notch.params.tracking_mode() == HarmonicNotchDynamicMode::Fixed)) {
```

Comment:
If we have a fixed notch, we still want to see it in the logs I think

Reply:
Seems a little redundant to log a parameter value at 25hz?